lms/djangoapps/discussion/tests/test_tasks.py

```diff
@@ -253,3 +254,41 @@ def test_subcomment_should_not_send_email(self):
 
     def test_second_comment_should_not_send_email(self):
         self.run_should_not_send_email_test(self.comment2)
+
+    @ddt.data((
+        {
+            'thread_id': 'dummy_discussion_id',
+            'thread_title': 'thread-title',
+            'thread_created_at': date.serialize(NOW),
+            'course_id': 'fake_course_edx',
+            'thread_author_id': 'a_fake_dude'
+        },
+        mock.patch('edx_ace.ace.send').start(),
+        {
+            'app_label': 'discussion',
+            'name': 'responsenotification',
+            'language': 'en',
+            'uuid': 'uuid1',
+            'send_uuid': 'uuid2',
+            'thread_id': 'dummy_discussion_id',
+            'thread_created_at': NOW
+        }
+    ))
+    @ddt.unpack
+    def test_track_notification_sent(self, context, message, test_props):
+
+        # Populate mock message
+        for key, entry in test_props.items():
+            setattr(message, key, entry)
+
+        with mock.patch('analytics.track') as mock_analytics_track:
+            _track_notification_sent(message, context)
+            self.assertEqual(mock_analytics_track.call_count, 1)
```

Comment:
you can use `assert_called_once_with` to assert that there is both exactly 1 call and it has the arguments you want